doc/build.md

```diff
@@ -291,3 +301,15 @@ Unit tests can be disable by adding -DDISABLE_TESTS=1 to CMake.
 ```shell
 -DDISABLE_TESTS=1
 ```
+
+### Embedded builds for RISC-V
+
+The libspdm libraries can be build along with mbedtls to target an embedded

```

Comment:
>The libspdm libraries can be built along with Mbed TLS to target an embedded
environment. The Integrator must provide a C library and runtime, such as Newlib.